mlkem/poly.c

```diff
@@ -656,3 +681,41 @@ void poly_sub(poly *r, const poly *a, const poly *b)
         r->coeffs[i] = a->coeffs[i] - b->coeffs[i];
     }
 }
+
+// REF-CHANGE: Duplicates fqmul() from ntt.c.
+// TODO: Consolidate this
```

Comment:
Yes please - which translation unit should this live in?

Reply:
Moved to `reduce.h` for now.